src/cascadia/TerminalSettingsModel/ActionAndArgs.cpp

```diff
@@ -119,6 +121,8 @@ namespace winrt::Microsoft::Terminal::Settings::Model::implementation
         { FindMatchKey, ShortcutAction::FindMatch },
         { TogglePaneReadOnlyKey, ShortcutAction::TogglePaneReadOnly },
         { NewWindowKey, ShortcutAction::NewWindow },
+        { IdentifyWindowKey, ShortcutAction::IdentifyWindow },
+        { IdentifyWindowsKey, ShortcutAction::IdentifyWindows },
```

Comment:
you could probably do a macro here, if you want

Reply:
Ooh spoiler alert: https://github.com/microsoft/terminal/compare/dev/migrie/f/rename-windows...dev/migrie/f/xmacro